.gitignore

```diff
@@ -0,0 +1,43 @@
+terraform-provider-jupiterone
```

Comment:
This `.gitignore` was adapted from other Terraform providers.

Reply:
I have a few file types that @fomentia would like to add to this file, so he can use it in all his projects.